docs/index.md

```diff
@@ -12,13 +12,13 @@ Information for [installing](Installation/Installation) and [upgrading](Installa
 Building the content of an Omeka site, including [items](Content/Items), [collections](Content/Collections), [tags](Content/Tags), and [item types](Content/Item_Types). The topics in this section correspond to options on the left-hand navigation of the Omeka admin dashboard.
 
 **Administration**  
-Managing your installation's [appearance](Admin/Appearance/Appearance_Settings/), [plugins](Admin/Adding_and_Managing_Plugins), [users](Admin/Users), and [site settings](Admin/Settings/General_Settings/); in Omeka, these options are only available to SuperUsers. The topics in this section correspond to the top navigation of the Omeka admin dashboard.
+Managing your installation's [appearance](Admin/Appearance/Appearance_Settings/), [plugins](Admin/Adding_and_Managing_Plugins), [users](Admin/Users), and [site settings](Admin/Settings/General_Settings/); in Omeka, these options are only available to SuperUsers. The topics in this section correspond to the top navigation of the Omeka administrative dashboard.
 
 **Plugins**  
-Documentation for RRCHNM produced plugins. If you cannot find documentation for a specific plugin here, please check its download page as documentation may be in a ReadMe or on a separate site. 
+Documentation for RRCHNM-produced plugins. The [Plugins directory on the website](https://omeka.org/classic/plugins/){target=_blank} includes many user-supplied plugins from our community; we have only documented plugins that we support. If you cannot find information for a specific plugin here, please [check on our registered plugins list](https://omeka.org/classic/plugins/){target=_blank}, or on its independent download page, as documentation may be in a ReadMe or on a separate site. 
```

Comment:
This should be Omeka-produced, not RRCHNM-produced.

Reply:
Or "plugins produced by the Omeka team" to match our usual terminology about this?

Reply:
That seems good. It's passive voice, but it's what we usually use.